src/service.py

```diff
@@ -15,3 +22,49 @@
 
 engine = create_engine(DATABASE_URL)
 SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+
+@contextmanager
+def get_db() -> Generator[Session]:
+    session = SessionLocal()
+    try:
+        yield session
+    finally:
+        session.close()
+
+
+def get_project(project_id: uuid.UUID) -> Projects | None:
+    with get_db() as session:
+        return session.query(Projects).get(project_id)
+
+
+def create_project_(project: Project) -> Projects:
+    with get_db() as session:
```

Comment:
there is a concept called dependency injection, you should pass it here and doesn't rely on session implementation